docs/Using Price Feeds/feed-registry.html

```diff
@@ -34,9 +34,6 @@
 ```solidity Mainnet
 latestRoundData(0x514910771AF9Ca656af840dff83E8264EcF986CA, 0x0000000000000000000000000000000000000348)
 ```
-```solidity Kovan
```

Comment:
should we remove the config from the page while kovan is still running?

Reply:
Restored